### PR TITLE
{tools} [{foss,intel}/2015a] GLPK 4.55 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**
Needs #1419 (GMP-6.0.0a-intel-2015a) Not anymore

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
missing dep: `GMP/6.0.0a-intel-2015a`

**Comment by boegel** [comment]
@hajgato: In our experience (@wpoely86 and myself), getting GMP v6.0.0a to work with `intel/2015a` is an issue (tests fail), so it's better to use the `GCC/4.9.2` build, and specify the `GMP` dep like:

```python
('GMP', '6.0.0a', '', ('GCC', '4.9.2'))
```

File: easybuild/easyconfigs/g/GLPK/GLPK-4.55-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ mixed integer programming (MIP), and other related problems.
+ It is a set of routines written in ANSI C 
+ and organized in the form of a callable library."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://ftp.gnu.org/gnu/%(namelower)s/']
+
+dependencies = [('GMP', '6.0.0a')]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GLPK/GLPK-4.55-foss-2015a.eb]: better use same dep as in the `intel-2015a` easyconfig?

```python
dependencies = [('GMP', '6.0.0a', '', ('GCC', '4.9.2'))]
```

**Comment by hajgato** [comment]
@boegel both packages using GMP compiled with GCC 4.9.2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7b5081480f509954ab4c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**